drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/referral_postings_loader.rb

```diff
@@ -303,24 +309,40 @@ def build_posting_records
           log_info "#{row.context} Skipping Referral #{referral_id} - no HoH"
           next
         end
-        hoh_mci_id = row_value(hoh_member_row, field: 'MCI_ID')
-        # 2) find the household_id for the project enrollment with that MCI
-        household_id = referral_household_id(hoh_mci_id, project_id)
-        # Household must be present for Accepted and Accepted Pending postings
+
         status = posting_status(row)
+        # 2) Find the household_id for the enrollment linked to this ReferralPosting row, if any.
+        household_id = nil
+        if status == accepted_status
+          # For Accepted referrals, the Enrollment ID should be present on the row.
+          enrollment_id = row_value(row, field: 'ENROLLMENTID', required: false)
+          unless enrollment_id.present?
```

Comment:
fwiw, row_value normalizes empty strings to nil so we don't have to deal with that case (`present?` isn't needed)